packages/nzbget/options.cpp.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/daemon/main/Options.cpp b/daemon/main/Options.cpp
+index f41bd82..6a3dc0e 100644
+--- a/daemon/main/Options.cpp
++++ b/daemon/main/Options.cpp
+@@ -182,10 +182,7 @@ const int BoolCount = 12;
+ const char* PossibleConfigLocations[] =
+ 	{
+ 		"~/.nzbget",
+-		"/etc/nzbget.conf",
+-		"/usr/etc/nzbget.conf",
+-		"/usr/local/etc/nzbget.conf",
+-		"/opt/etc/nzbget.conf",
++		"/data/data/com.termux/files/usr/etc/nzbget.conf",
```

Comment:
Please use @TERMUX_PREFIX@ instead of hard-coding /data/data/com.termux/files/usr, it simplifies if someone wants to install termux into another prefix. @TERMUX_PREFIX@ is sed'ed in build-package.sh and replaced with the actual path.

Reply:
Didn't realize that. I fixed that. Thanks.